### ISSUE TITLE
[Appeal] (Help) How to fix Connection close after QR code scan?

### ISSUE THREAD

**Issue by DineshValor** [Diegoson/X-Astral#1]
I'm working to fix this [Bot](https://github.com/DineshValor/valor/blob/bot/lib/connection.js)

Came back after 2 year and its failed to connect 🥲, need helping hands...

Can you help me?

![Screenshot_20240825-195929](https://github.com/user-attachments/assets/5ade8701-4143-4d2f-938e-326e75b871ef)

**Comment by Diegoson** [comment]
Have you tried to change baileys version?

**Comment by DineshValor** [comment]
> Have you tried to change baileys version?

Yep, updated baileys version in package.json and also edited mandatory changes in scripts.

But still I don't get it, as soon as scan the code its an error.

**Comment by Diegoson** [comment]
Your bot script bruh way too confusing

**Comment by DineshValor** [comment]
> Your bot script bruh way too confusing

Everything related to connection, script located in /lib/connection.js
https://github.com/DineshValor/valor/blob/bot/lib/connection.js
